templates/zerver/help/reply-to-message.md

```diff
@@ -0,0 +1,58 @@
+# Replying to a Message
```

Comment:
I'm not sure the title should be an `-ing` noun, some titles start with verbs, some with nouns. Was there any discussion on that? It would be nice if they were more consistent.